src/webui/www/private/views/rss.html

```diff
@@ -697,11 +697,12 @@
         };
 
         const removeItem = (paths) => {
+            let encodePaths = paths.map(function(path) { return encodeURIComponent(path); });
```

Comment:
Use `const` and I would name it `encodedPaths`

Reply:
Done.